dokku

```diff
@@ -2,7 +2,8 @@
 set -eo pipefail
 shopt -s nullglob
 
-export DOKKU_ROOT=${DOKKU_ROOT:=~dokku}
+export DOKKU_USER=${DOKKU_USER:=$(cat "/home/.dokku-user")}
```

Comment:
Yeah so this I wanted to talk with you guys about. I can't think of a good way to go about this in a clean manner. What do you suggest?